utils/display/flink_jobs.go

```diff
@@ -51,3 +50,89 @@ func FlinkJobsTable(flinkJobs []*meroxa.FlinkJob) string {
 func PrintFlinkJobsTable(jobs []*meroxa.FlinkJob) {
 	fmt.Println(FlinkJobsTable(jobs))
 }
+
+func FlinkJobTable(flinkJob *meroxa.FlinkJob) string {
+	mainTable := simpletable.New()
+	mainTable.Body.Cells = [][]*simpletable.Cell{
+		{
+			{Align: simpletable.AlignRight, Text: "UUID:"},
+			{Text: flinkJob.UUID},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Name:"},
+			{Text: flinkJob.Name},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Created At:"},
+			{Text: flinkJob.CreatedAt.String()},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Updated At:"},
+			{Text: flinkJob.UpdatedAt.String()},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Input Streams:"},
+			{Text: strings.Join(flinkJob.InputStreams, ", ")},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Output Streams:"},
+			{Text: strings.Join(flinkJob.OutputStreams, ", ")},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Lifecycle State:"},
+			{Text: string(flinkJob.Status.LifecycleState)},
+		},
+	}
+	if flinkJob.Status.State != "" {
+		mainTable.Body.Cells = append(mainTable.Body.Cells, []*simpletable.Cell{
+			{Align: simpletable.AlignRight, Text: "Job State:"},
+			{Text: string(flinkJob.Status.State)},
+		})
+	}
+
+	if flinkJob.Status.ReconciliationState != "" {
```

Comment:
These guard checks are helpful for filtering out empty values, I'll keep this in mind for future CLI work as well